src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrTopicOverviewTaskTest.java

```diff
@@ -0,0 +1,148 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mserver.crawler.basic.TopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.WdrTopicUrlDTO;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import static org.hamcrest.CoreMatchers.equalTo;
+import org.hamcrest.Matchers;
+import org.jsoup.Jsoup;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+@PowerMockRunnerDelegate(Parameterized.class)
+@PowerMockIgnore("javax.net.ssl.*")
+public class WdrTopicOverviewTaskTest extends WdrTaskTestBase {
+
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] {  
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)